src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Item{
+    private final String itemType = "[D]";
```

Comment:
Since itemType is a final variable similar to a constant, would it be better to use the constant naming convention?